wiki/Tournaments/Zotcade/2019/en.md

```diff
@@ -255,22 +306,183 @@ The iDOL@NSTER is run by various osu! community members at UCI or visiting for t
 - Tiebreaker
   - **[MC Mental - MC MENTAL @ HIS BEST (hi19hi19) [Hard]](https://osu.ppy.sh/beatmapsets/503710#mania/1071695)**
 
-## Ruleset
+## Match Results
+
+### osu!
+
+#### Grand Finals
+
+| Player 1 |  |  | Player 2 | Notes |
+| --: | :-: | :-: | :-- | :-- |
+| **fieryrage** ![][flag_US] | **6** | 1 | ![][flag_US] Mathyu | [Multiplayer match](https://osu.ppy.sh/community/matches/51947819) |
```

Comment:
I would change the Multiplayer Match label to just `#1` to follow the rest of the articles
For the rest of the notes without mp links would be better to just lowercase those and put dashes?, like -forfeit-

Reply:
ok
what does the #1 even mean tho i didn't understand so i just wrote multiplayer match

Reply:
is a reference to the first link, in some matches for mostly bancho reasons, matches needs to be resumed on another room, the current notation for this is `#1 #2 ... #N` and has been like that since the dawn of time.

An example can be seen here on this TWC 2018 article

![](https://walter.s-ul.eu/i9lYTFTl.png)

Reply:
should i rename column to "match link " or somethn then

Reply:
Yeah, probably better.